grails-app/services/org/bbop/apollo/OrganismService.groovy

```diff
@@ -76,4 +83,62 @@ class OrganismService {
         }
 
     }
+
```

Comment:
I might move this into a more general utility service (ZipService, or FileService) as nothing specific to organisms.